src/index.jsx

```diff
@@ -20,7 +22,18 @@ subscribe(APP_READY, () => {
   ReactDOM.render(
     <AppProvider>
       <Header />
-      <ExamplePage />
+      <Switch>
+        <Route
+          path="/proctored-exam-settings/:id"
```

Comment:
I feel like naming this `course_id` would more clearly describe what this value is.